Tidy up Stylelint errors

Currently, a Stylelint error is presented like so:

```
[20:41:24] 

assets/css/frontend/style.css
 29:14  ✖  Unexpected named color "red"   color-named



[20:41:24] 'lint-css' errored after 342 ms
[20:41:24] Error in plugin "gulp-stylelint"
Message:
    Failed with 1 error
Details:
    domainEmitter: [object Object]
    domain: [object Object]
    domainThrown: false
```

I think we should suppress most of this information, at least while watching the files. The only actual useful information is:
```
assets/css/frontend/style.css
 29:14  ✖  Unexpected named color "red"   color-named
```

The rest is pure noise.

@dainemawer In my experience, running `gulp watch --silent` is good enough to hide most of this stuff.

That returns just the following:
```
[17:21:34] 

assets/css/frontend/style.css
 29:2  ✖  Expected variable for "color".   sh-waqar/declaration-use-variable
 29:9  ✖  Unexpected named color "red"     color-named
```

Webpack gives us a bit more:
```
[17:21:56] Version: webpack 4.27.0
Built at: 12/04/2018 5:21:56 PM
 6 assets
Entrypoint admin = admin.min.js admin.min.js.map
Entrypoint frontend = frontend.min.js frontend.min.js.map
Entrypoint shared = shared.min.js shared.min.js.map

ERROR in ./assets/js/admin/admin.js
Module Error (from ./node_modules/eslint-loader/index.js):

/Users/jodi/Projects/10up/wp-local-docker/theme-scaffold-gutenberg-test/wordpress/wp-content/themes/theme-scaffold/assets/js/admin/admin.js
  3:7  error  'a' is assigned a value but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

The main thing is that it hides the gulp error noise.

Alternatively, we could set a silent flag in a `gulp.json` settings file in the root, but I don't think we'd want to suppress errors for all gulp processes.

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "start": "npm install && gulp",
-    "watch": "gulp watch",
+    "watch": "gulp watch --silent",
     "build": "gulp",
     "build-release": "npm install && composer install --no-dev -o && gulp",
     "lint-release": "npm install && composer install && npm run lint",

